### ISSUE TITLE
DOB for patients

### ISSUE THREAD

**Issue by jaredleitner** [MIT-LCP/mimic-code#637]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

In the MIMIC III clinical database description it states:

"Dates of birth for patients aged over 89 were shifted to obscure their true age and comply with HIPAA regulations: these patients appear in the database with ages of over 300 years."

After looking at the PATIENTS table, I find all the DOBs are shifted. I understand the DOBs for patients over the age of 89 have been protected, however, where can I find the true DOBs for patients younger than 89?

**Comment by alistairewj** [comment]
You are not allowed to access the true date of birth. Instead, you have a date shifted version. This date shift is consistent across all times for the patient, allowing you to calculate their age as the time of their ICU admission minus their date of birth, as is done in the following script.

https://github.com/MIT-LCP/mimic-code/blob/240823295650c5933260aa7d37665a7caa2c55df/concepts/demographics/icustay-detail.sql#L24

**Comment by ZhiliangWu** [comment]
> You are not allowed to access the true date of birth. Instead, you have a date shifted version. This date shift is consistent across all times for the patient, allowing you to calculate their age as the time of their ICU admission minus their date of birth, as is done in the following script.
> 
> https://github.com/MIT-LCP/mimic-code/blob/240823295650c5933260aa7d37665a7caa2c55df/concepts/demographics/icustay-detail.sql#L24

@alistairewj Does it also mean that the age information for patients older than 89 are removed through this shift process. I.e. we can only know they are older than 89, but no more information of the exact age like 92, 93.

**Comment by alistairewj** [comment]
Yes, as mentioned here https://mimic.physionet.org/mimicdata/time/ - ages over 89 are shifted such that the patient age appears to be 300 in the database.

**Comment by ArianKhorasani** [comment]
I have an issue with finding ( or calculating ) the ages of patients who are younger than 10 years old! any suggestions?

**Comment by ArianKhorasani** [comment]
Dear @alistairewj - I have an issue with calculating ages of patients who are younger than 10 years old. I understand that there is no access to the true data of birth, and there is a date shifted version. but If I'm not mistaken, this consideration is for patients older than 89 years old. what is your suggestion? Also, I do have access to the whole MIMIC III's tables in CSV format. Thanks!

**Comment by alistairewj** [comment]
MIMIC does not have pediatric patients. MIMIC-III has neonates (age ~ 0 years), and MIMIC-IV does not.

**Comment by ArianKhorasani** [comment]
> MIMIC does not have pediatric patients. MIMIC-III has neonates (age ~ 0 years), and MIMIC-IV does not.

@alistairewj I see. Any thoughts or Suggestions on how can I find the neonates (age ~ 0 years) ages on MIMIC III by using Python? I just want to collect data of Neonates ( less than 0 years age ) for my project.

**Comment by alistairewj** [comment]
For MIMIC-III you can calculate admittime - dob, and anything < 1 year is a neonate. Typically it is 0 days as they are "admitted" when they are born.

**Comment by ArianKhorasani** [comment]
> Member



> For MIMIC-III you can calculate admittime - dob, and anything < 1 year is a neonate. Typically it is 0 days as they are "admitted" when they are born.

Dear @alistairewj - I did exactly what you suggested, Thanks! but just another question, is it normal that the Neonates' vital signs ( only Blood Pressure and Oxygen Saturation ) doesn't have any values in columns? I did extract vital signs ( blood pressure, Oxygen Saturation, Respiratory rate, Temperature, and Heart rate ), but only Blood Pressure and Oxygen Saturation don't have any rows and the CSV files are empty. but the rest of the vital signs data are OK! Just wondering.
